.github/pull_request_template.md

```diff
@@ -1,11 +1,25 @@
 ## Issue: <!-- link the issue or issues this PR resolves here -->
-<!-- If your PR depends on changes from another pr link them here and describe why they are needed on your solution section. -->
- 
+<!-- If your PR depends on changes from other PRs, link them here and describe why they are needed for your solution section. -->
+
 ## Problem
-<!-- Describe the root cause of the issue you are resolving. This may include what behavior is observed and why it is not desirable. If this is a new feature describe why we need this feature and how it will be used. -->
+<!-- Describe the root cause of the issue you are resolving. 
+This may include what behavior is observed and why it is not desirable. If this is a new feature, describe why we need it and how it will be used. -->
  
```

Comment:
```suggestion

```

Remove space here

Reply:
I wanted to leave a space so that you have a location to start writing when filling out the template. Let me know if you think this is unconventional.

Reply:
Do you mean a space or a newline? I think a newline makes sense as you said. That line isn't empty, it has a space in it, which we probably don't need. It's all nitpicks anyway so feel free to leave as is :+1:

Reply:
Ahh, good catch. I thought you were talking about the new line.